lib/drives/pages/create_drive_page.dart

```diff
@@ -240,7 +240,7 @@ class _CreateDriveFormState extends State<CreateDriveForm> {
               ),
             ],
           ),
-          const Divider(),
+          const SizedBox(height: 5),
```

Comment:
```suggestion
          const SizedBox(height: 10),
```
then it's the same as the others. 
Same for search